daprdocs/content/en/operations/components/component-scopes.md

```diff
@@ -84,3 +120,8 @@ scopes:
 ## Example
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/8W-iBDNvCUM?start=1763" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Related links
+- [How-To: Configure Pub/Sub components with multiple namespaces]({{< ref "pubsub-namespaces.md" >}}) 
+- [How To: Use secret scoping]({{< ref "secrets-scopes.md" >}}) 
```

Comment:
"How To" or "How-To"?

Reply:
Third option: Drop the how-to altogether

Reply:
Fixed